opencga-catalog/src/main/java/org/opencb/opencga/catalog/managers/UserManager.java

```diff
@@ -590,7 +590,7 @@ public OpenCGAResult<User> update(String userId, ObjectMap parameters, QueryOpti
 
             userId = getValidUserId(userId, payload);
             for (String s : parameters.keySet()) {
-                if (!s.matches("name|email|organization|attributes")) {
+                if (!s.matches("name|email|attributes")) {
```

Comment:
I added "this" line 10 years ago... what a horrible line...
https://github.com/opencb/opencga/commit/f51494c15d010625959c7cebaa2b5244436eea68#diff-ae76d8c01706b07219bbea60e7e513295a883a4bb9b3434a1f24897345cf4cd4R253-R257

Reply:
The day I completely change it you'll miss it.